### PR TITLE
spotify issue #138 : make Docker Host available in DockerClient

### PR DESCRIPTION
**Pull request by mbaechler**
Could you consider merge this simple feature ?

### CODE REVIEW

File: src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class DefaultDockerClientUnitTest {
+
+  @Test
+  public void testHostForUnixSocket() {
+    DefaultDockerClient client = DefaultDockerClient.builder().uri(DefaultDockerClient.DEFAULT_UNIX_ENDPOINT).build();
```
Comment (davidxia) [inline src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java]: Can we wrap these lines so they don't go over 100 chars?

**Comment by davidxia** [comment]
@mbaechler :+1: Just one nitpick above.

File: src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java (hunk 1 above)
Comment (mbaechler) [inline src/test/java/com/spotify/docker/client/DefaultDockerClientUnitTest.java]: Should be ok now

**Comment by mbaechler** [comment]
The PR is up to date, could you please merge it ?

**Pull request merged**